src/gui/torrentcreatordlg.ui

```diff
@@ -224,6 +224,13 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QCheckBox" name="checkPreserveFileOrder">
+        <property name="text">
+         <string>Preserve file order</string>
```

Comment:
I think it's better to use "Optimize alignment" and by default enabled.

Also we should encourage this flag usage (favor optimize alignment) and unless users have good reason to disable this, the user will keep it on.